docs/development/tls-setup.md

```
@@ -0,0 +1,156 @@
+| File | Description | Permissions |
+|------|-------------|-------------|
+| `ca.pem` | Self-signed CA certificate | 644 |
+| `node.pem` | Node certificate (signed by CA) | 644 |
+| `node-key.pem` | Node private key | 600 |
+| `ssl_dist.conf` | Erlang ssl_dist configuration | 644 |
+
+### Certificate Location
+
+Certificates are stored under the workspace's TLS directory:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)